functests/2_performance_update/updating_profile.go

```diff
@@ -35,6 +35,8 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 	chkCmdLine := []string{"cat", "/proc/cmdline"}
 	chkKubeletConfig := []string{"cat", "/rootfs/etc/kubernetes/kubelet.conf"}
 	chkIrqbalance := []string{"cat", "/rootfs/etc/sysconfig/irqbalance"}
+	chkDefaultSMPAffinity := []string{"cat", "/proc/irq/default_smp_affinity"}
```

Comment:
@fromanirh please see existing naming.

Reply:
sure, let's fix all of them (in another future PR of course).

Reply:
ack